src/database/migrations/20241217110259-update-reports-data.js

```diff
@@ -0,0 +1,293 @@
+'use strict'
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+	async up(queryInterface, Sequelize) {
+		// Insert the report data into the reports table
+		await queryInterface.bulkInsert('reports', [
+			{
+				code: 'total_number_of_sessions_attended',
+				title: 'Total number of sessions attended',
+				description: 'Total sessions attended by user in big number',
+				report_type_id: 1,
```

Comment:
@sumanvpacewisdom we should map to report type title i think, title should be primary key in report type